.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/unit/kpatch-unit-objs"]
+	path = test/unit/kpatch-unit-objs
+	url = https://github.com/sm00th/kpatch-unit-objs.git
```

Comment:
This is just for reference, should probably be switched to ad dynup repo before merging.